packages/garbo/src/config.ts

```diff
@@ -98,6 +99,14 @@ function defaultTarget() {
 export const targettingMeat = () =>
   $monsters`cheerless mime executive`.includes(globalOptions.target);
 
+export const targettingItems = () =>
+  valueDrops(globalOptions.target) > meatDrop(globalOptions.target);
+
+export const goosoDroneEligible = () =>
+  targettingItems() &&
+  itemDropsArray(globalOptions.target).length === 1 &&
+  have($familiar`Grey Goose`);
```

Comment:
Export this from `lib`, and use the word `goose` instead of `gooso` everywhere possible in this PR